modules/oxide-components/src/main/ts/keynav/keyboard/Keys.ts

```diff
@@ -0,0 +1,23 @@
+const TAB: ReadonlyArray<number> = [ 9 ];
+const ENTER: ReadonlyArray<number> = [ 13 ];
+const SHIFT: ReadonlyArray<number> = [ 16 ];
+const END: ReadonlyArray<number> = [ 35 ];
+const LEFT: ReadonlyArray<number> = [ 37 ];
+const UP: ReadonlyArray<number> = [ 38 ];
+const RIGHT: ReadonlyArray<number> = [ 39 ];
+const DOWN: ReadonlyArray<number> = [ 40 ];
+const SPACE: ReadonlyArray<number> = [ 32 ];
+const ESCAPE: ReadonlyArray<number> = [ 27 ];
```

Comment:
The value doesn't have to be an array of one number. We should look into changing it

Reply:
I agree I think the reason we have an array of number is because this implementation was ported straightly from Alloy. I think we should start using the supported [key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) property instead of the [deprecated keyCode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) property. What do you think ?

Reply:
Yes. Please ensure that `key` is supported by all browsers we support.

Reply:
If you look at the mdn website, it says it is supported across all browsers, even the mobile ones.

Reply:
There's an open ticket to use `key`, I think. It's long been the supported path.

These are arrays because once we add shortcuts that need to use the command or windows key, there are 3 we need to check
https://github.com/tinymce/tinymce/blob/ca9a89cc67cf8ab4eeab3cc11c74c947eb63cdd3/modules/alloy/src/main/ts/ephox/alloy/alien/Keys.ts#L20

Reply:
ah that was part of the holdup, thank you for the reminder

Reply:
So what should we do in this case ? Add a TODO comment and Jira ticket to update agar ?

Reply:
yes

Reply:
Any time you see `done: shelved` it's 100% available to reopen. That process was just a big cleanup we did a couple of years ago.